src/main/game_info.cpp

```diff
@@ -23,12 +23,30 @@
 #include <AppPath/app_path.h>
 #include <IniProcessor/ini_processing.h>
 #include "game_info.h"
+#include "cheat_code.h"
 
 #include "../version.h"
 
 
 GameInfo g_gameInfo;
 
+static void readCheats(IniProcessing &conf, std::vector<GameInfo::CheatAlias> &dst, const std::string &group)
+{
+    dst.clear();
+    conf.beginGroup(group);
+    {
+        auto ak = conf.allKeys();
+        for(const auto &k : ak)
+        {
+            GameInfo::CheatAlias ca;
+            ca.first = k;
+            conf.read(k.c_str(), ca.second, std::string());
+            dst.push_back(ca);
```

Comment:
I'm worried that the alias could be an empty string. I assume that elsewhere you check for empty aliases? Or is the default is necessarily unused since k was in the keys array?

Reply:
Ah, damn, I should just skip adding this if becomes empty...

Reply:
Fixed